tests/nest_tests/resources/vectors_test_synapse.nestml

```diff
@@ -0,0 +1,24 @@
+# vectors_test_synapse
+# ####################
+#
```

Comment:
Could you add the full copyright header? (We should add a check for this in the CI...)